crates/weaver_forge/src/config.rs

```diff
@@ -304,7 +308,22 @@ impl CaseConvention {
             CaseConvention::CamelCase => text.to_case(Case::Camel),
             CaseConvention::SnakeCase => text.to_case(Case::Snake),
             CaseConvention::ScreamingSnakeCase => text.to_case(Case::ScreamingSnake),
-            CaseConvention::KebabCase => text.to_case(Case::Kebab),
+            CaseConvention::KebabCase => {
+                // Convert to kebab case but do not consider digits
+                // as boundaries. So that `k8s` will stay `k8s` and
+                // not `k-8-s`.
+                let conv = KEBAB_CASE.get_or_init(|| {
+                    Converter::new()
+                        .add_boundary(Hyphen)
+                        .remove_boundary(DigitLower)
```

Comment:
Could you add this fix to LowerCase + TitleCase too?

Reply:
Done -> https://github.com/open-telemetry/weaver/pull/106/commits/afdf547f705fae69d5b4c751fdb3998100d352a9